### PR TITLE
Add 21.09 release announcement

### PR DESCRIPTION
**Pull request by mvdbeek**
Also kudos @NickSto @dannon, this is a really smooth editing process.

### CODE REVIEW

File: content/news/2021-11-galaxy-release-21-09/index.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+Galaxy is now using [SQLAlchemy](https://www.sqlalchemy.org/) version 1.4,
+which is a prerequisite for asynchronously interacting with the Database. We
+also modernized the way our Database models are defined to the more commonly
+used Declarative Mapping approach, which is more concise and better
+documented.
+
+### Modernization of Tool Form interface
+
+The Tool Form interface has been almost completely migrated from
```
[the remark this answers is not in the document]
Comment (beatrizserrano) [reply content/news/2021-11-galaxy-release-21-09/index.md:73]: no sure about this capitalization/lower cases, please ignore if it's wrong

File: content/news/2021-11-galaxy-release-21-09/index.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+[Backbone](https://backbonejs.org) to [Vue](https://vuejs.org/).
+This improves the reactivity of parameter validation and enables the migration of
+Tool parameters to Vue, which will also allow us to add many more types of
+Tool parameters. We further migrated the Tool Form variant for the Workflow
+Editor, the Workflow Run Form and the Show Dataset Parameter page to Vue.
```
Comment (beatrizserrano) [inline content/news/2021-11-galaxy-release-21-09/index.md:78]:
```suggestion
Editor, the Workflow run form and the Show Dataset Parameter page to Vue.
```

File: content/news/2021-11-galaxy-release-21-09/index.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+### New User Welcome Page
+
+We have added a Page that new users will be directed to after creating a new Galaxy account.
```
Comment (beatrizserrano) [inline content/news/2021-11-galaxy-release-21-09/index.md:82]:
```suggestion
We have added a page that new users will be directed to after creating a new Galaxy account.
```

File: content/news/2021-11-galaxy-release-21-09/index.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/XcZgQbGojh8" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+### Reports
+
+Report components used to try and arrange themselves in a smart way, but due to some issues with the floating of report components these will now be full width until we can figure out a good way to provide a similar feature allowing you to control the layout of the report ([Pull Request 12085](https://github.com/galaxyproject/galaxy/pull/12085)).
+
+![A screenshot of the reports interface is shown, there are a number of 'report components' like table previews embedded and these are full width across the report.](images/21.09-reports.png)
+
+### Enhancements for working with remote data and distributed computing resources
```
Comment (beatrizserrano) [inline content/news/2021-11-galaxy-release-21-09/index.md:44]:
```suggestion
### Enhancements for Working with Remote Data and Distributed Computing Resources
```

File: content/news/2021-11-galaxy-release-21-09/index.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+added an ``Export datasets`` Tool for exporting individual Datasets and Dataset
+collections to configured remote locations. The Tool will automatically
+maintain the name and structure of Datasets and Dataset Collections. We have
+added the possibility to import entire folders of remote data and made the
+data selection dialog more intuitive. We also added numerous improvements and
+fixes to the extended metadata collection strategy and [Pulsar](https://github.com/galaxyproject/pulsar),
+so that many more Tools are able to run in [Pulsar](https://github.com/galaxyproject/pulsar)
+and write data back to storage without passing through Galaxy first.
+
+### Migration to FastAPI and extended Documentation of API routes
```
Comment (beatrizserrano) [inline content/news/2021-11-galaxy-release-21-09/index.md:58]:
```suggestion
### Migration to FastAPI and Extended Documentation of API Routes
```

File: content/news/2021-11-galaxy-release-21-09/index.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+
+### Migration to SQLAlchemy 1.4 and Declarative Mapping
+
+Galaxy is now using [SQLAlchemy](https://www.sqlalchemy.org/) version 1.4,
+which is a prerequisite for asynchronously interacting with the Database. We
+also modernized the way our Database models are defined to the more commonly
+used Declarative Mapping approach, which is more concise and better
+documented.
+
+### Modernization of Tool Form interface
```
Comment (beatrizserrano) [inline content/news/2021-11-galaxy-release-21-09/index.md:71]:
```suggestion
### Modernization of Tool Form Interface
```

File: content/news/2021-11-galaxy-release-21-09/index.md [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+
+### Importing Data
+
+Selecting datasets from the Remote Files view has gotten easier! Now you can select folders and files, and it will import all of the datasets recursively in those folders. Previously you could only select files within a folder so this is a huge improvement in usability of such a key new feature of Galaxy ([Pull Request 12310](https://github.com/galaxyproject/galaxy/pull/12310)). But with great power comes great responsibility, be careful not to import the entirety of NCBI!
+
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/XcZgQbGojh8" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+### Reports
+
+Report components used to try and arrange themselves in a smart way, but due to some issues with the floating of report components these will now be full width until we can figure out a good way to provide a similar feature allowing you to control the layout of the report ([Pull Request 12085](https://github.com/galaxyproject/galaxy/pull/12085)).
```
Comment (beatrizserrano) [inline content/news/2021-11-galaxy-release-21-09/index.md:40]:
```suggestion
Report components used to automatically arrange themselves in a smart way, but due to some issues with the floating of components, these will now be full width until we can figure out a good way to provide a similar feature, allowing you to control the layout of the report ([Pull Request 12085](https://github.com/galaxyproject/galaxy/pull/12085)).
```

Comment (beatrizserrano) [reply content/news/2021-11-galaxy-release-21-09/index.md:40]: `allowing you to control the layout of the report` is a bit confusing to me: Does it mean that in the future users will be able to arrange the components or is it happening already?

**Review by beatrizserrano — COMMENTED** [review]
thank you @mvdbeek!

File: content/news/2021-11-galaxy-release-21-09/index.md (hunk 7 above)
Comment (mvdbeek) [reply content/news/2021-11-galaxy-release-21-09/index.md:40]: In the future, I think we can just drop that comment.

**Review by beatrizserrano — APPROVED** [review]

**Pull request merged**